nettacker/locale/ka.yml

```
@@ -0,0 +1,122 @@
+send_unsuccess_event_from_module: "عمل-{0}|{1}|{2}|موڈیول-تھریڈ {3}/{4}|درخواست-تھریڈ {5}/{6}| سبھی شرط ناکام گئیں"
+sending_module_request: "عمل-{0}|{1}|{2}|موڈیول-تھریڈ {3}/{4}| درخواست بھیجوانہ {5} از {6}"
+verbose_event: ہر تھریڈ ہند حالت دیکھنہ خاطر verbose ایوینٹ آن کرنہ
+no_events_for_report: رپورٹ تیار کرنہ خاطر کوئی ایوینٹس موجود نہ چھ! ییہ حصہ ترک کرنہ.
+set_hardware_usage: سکین دوران ہارڈویئر استعمال طے کرنہ۔ (low, normal, high, maximum)
+show_all_modules: سبھی موڈیولس تہ اُنکی معلومات دیکھاو
+show_all_profiles: سبھی پروفائلز تہ اُنکی معلومات دیکھاو
+single_process_started: عمل-{0}| عمل شروع گئ!
+software_version: سافٹ ویئر ورژن دیکھاو
+start_api_server: API سروس شروع کرنہ
+start_multi_process: {0} اہداف {1} عملس منز شامل کرنہ.
+start_parallel_module_scan: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)